modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -76,13 +76,19 @@ public function reregister_menu_items() {
 			$this->add_site_card_menu( $domain );
 		}
 
-		/*
+		/**
 		 * Whether links should point to Calypso or wp-admin.
 		 *
+		 * Options:
 		 * true  - Calypso.
 		 * false - wp-admin.
+		 *
+		 * @module masterbar
+		 * @since 9.3.0
+		 *
+		 * @param bool $calypso Whether menu item URLs should point to Calypso.
 		 */
-		$calypso = true;
+		$calypso = apply_filters( 'jetpack_admin_menu_calypso_links', true );
```

Comment:
```suggestion
		$calypso = apply_filters( 'jetpack_admin_menu_use_calypso_links', true );
```

Tiny change helps to make filter a little more self documenting. Not sure if it meets the standards though 🤷‍♂️ .

Reply:
I like it too. 👍